### PR TITLE
3.2.1 release

### PR DESCRIPTION
**Pull request by wing328**
### PR checklist

- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [ ] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `3.3.x`, `4.0.x`. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

(details of the change, additional tests that have been done, reference to the issue for tracking, etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -485,7 +485,7 @@ Here are some companies/projects (alphabetical order) using OpenAPI Generator in
...
 - 2018/06/21 - [Connect your JHipster apps to the world of APIs with OpenAPI and gRPC](https://fr.slideshare.net/chbornet/jhipster-conf-2018-connect-your-jhipster-apps-to-the-world-of-apis-with-openapi-and-grpc) by [Christophe Bornet](https://github.com/cbornet) at [JHipster Conf 2018](https://jhipster-conf.github.io/)
-- 2018/06/27 - [Lessons Learned from Leading an Open-Source Project Supporting 30+ Programming Languages](https://speakerdeck.com/wing328/lessons-learned-from-leading-an-open-source-project-supporting-30-plus-programming-languages) - [William Cheng](https://github.com/wing328) at [LinuxCon + ContainerCon + CloudOpen China 2018](https://www.lfasiallc.com/events/lc3-2018/)
+- 2018/06/27 - [Lessons Learned from Leading an Open-Source Project Supporting 30+ Programming Languages](https://speakerdeck.com/wing328/lessons-learned-from-leading-an-open-source-project-supporting-30-plus-programming-languages) - [William Cheng](https://github.com/wing328) at [LinuxCon + ContainerCon + CloudOpen China 2018](https://www.lfasiallc.com/events/lc3.2.28/)
```
Comment (wing328) [inline README.md]: Seems like regular expression issue in the update script. I'll roll it back manually for the time being.

cc @jmini

**Comment by jmini** [comment]
@wing328 just created the release tag: 
https://github.com/OpenAPITools/openapi-generator/releases/tag/v3.2.1

**Comment by jmini** [comment]
Follow up PR: https://github.com/OpenAPITools/openapi-generator/pull/810

**Comment by jmini** [comment]
Check script: `ruby bin/utils/release_checkout.rb 3.2.1`

Output:
```
Running checkout on OpenAPI Generator release 3.2.1
Checking homebrew forumla ... [ERROR]
> https://raw.githubusercontent.com/Homebrew/homebrew-core/master/Formula/openapi-generator.rb not yet updated with https://search.maven.org/remotecontent?filepath=org/openapitools/openapi-generator-cli/3.2.1/openapi-generator-cli-3.2.1.jar
Checking openapi-generator JAR ... [OK]
Checking openapi-generator-cli JAR ... [OK]
Checking openapi-generator-maven-plugin JAR ... [OK]
Checking openapi-generator-gradle-plugin JAR ... [ERROR]
> http://central.maven.org/maven2/org/openapitools/openapi-generator-gradle-plugin/3.2.1/openapi-generator-gradle-plugin-3.2.1.jar not found
Checking openapi-generator-online JAR ... [OK]
Checking openapi-generator-project pom.xml ... [OK]
Checking openapi-generator README.md ... [OK]
Checking openapi-generator-online docker ... [OK]
Checking openapi-generator-cli docker ... [OK]
```

**Comment by jmini** [comment]
Milestone [`3.2.1`](https://github.com/OpenAPITools/openapi-generator/milestone/10) closed

**Comment by jmini** [comment]
Master is now `3.2.2-SNAPSHOT`

**Comment by jmini** [comment]
Gradle Plugin is published on Maven Central:
https://repo.maven.apache.org/maven2/org/openapitools/openapi-generator-gradle-plugin/3.2.1/

**Comment by jmini** [comment]
Homebrew: Homebrew/homebrew-core#31124

**Pull request merged**